pkg/provisioner/k8sprovisioner.go

```diff
@@ -18,7 +18,13 @@ var listOptionsMatchLabels = metav1.ListOptions{
 }
 
 var podInitCloneArgs = []string{"git", "clone"}
-var podContainerListArgs = []string{"/bin/sh", "-c", "ls -alh"}
+var podContainerListArgs = []string{"/bin/sh", "-c", `apt-get update && \
+apt-get install -y npm && \
+make deps && \
+make swag-force && \
+go install && \
+cd test && \
+wharf-cmd run -s test --loglevel debug`}
```

Comment:
If its needed for testing, I think it could stay. Doesn't matter much what it is since we don't have a real implementation is.

However `ls -alh` is obviously just for testing, while this looks like a real implementation. Need some good `// TODO: ...` comments here to show that this is not here to stay.

Same for your other "just for testing" review threads

Reply:
Added todo comment.